### PR TITLE
Random subpsace method

### PR DESCRIPTION
**Pull request by rguo123**

### CODE REVIEW

File: multiview/construct/rsm.py [hunk 1]
```diff
@@ -46,25 +49,26 @@ def random_subspace_method(X, n_features=None, n_views=1):
...
+
     for _ in range(n_views):
         view = np.copy(X)
         features_selected = []
-        
+
         while len(features_selected) != n_features:
-            
+
             feature = random.randint(0, cols - 1)
```
Comment (gmischl1) [inline multiview/construct/rsm.py]: Should there be a parameter to allow the user to choose the seed for this function?

File: multiview/construct/rsm.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        Number of views to construct.
+
+    Returns
+    -------
+    views : list of array-like matrices
+        List of constructed views (each matrix has shape [n_rows, n_cols]).
+
+    """
+
+    # todo let max_features be float for probability
```
Comment (rflperry) [inline multiview/construct/rsm.py]: Merge fine if this todo is a future todo

File: multiview/construct/rsm.py (hunk 1 above)
Comment (rflperry) [reply multiview/construct/rsm.py]: I think it sufficient to set seed before calling the function

File: multiview/construct/rsm.py (hunk 2 above)
Comment (rguo123) [reply multiview/construct/rsm.py]: Ooh good catch. Already implemented just forgot to delete comment.

**Pull request merged**